web/src/components/Errors/index.css

```diff
@@ -0,0 +1,8 @@
+.server-error{
```

Comment:
Don't create a separate file for the component. Use `styles/global.css` file to add this CSS code.